### DISCUSSION TITLE
对视频弹窗的折中处理

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1863] [功能建议 / Ideas]
考虑到部分视频弹窗有时候还是必要的（如删除投票弹窗就只能看着屏幕上的一堆的 `1` `2` 弹幕一脸懵逼），能否提供一些选项来降低视频弹窗的干扰性？
如增大透明度，光标悬停在弹窗上时再恢复；缩小弹窗；进一步的如自定义弹窗位置。
针对投票弹窗，能否提取信息然后以一种更简洁些方式展示在屏幕顶部居中（或者自定义？）

**Comment by the1812** [comment]
弹窗应该有办法改改, 但投票导致的一堆 `1` 和 `2` 也许 pakku 更擅长处理

**Reply by Geeyun-JY3** [reply]
一堆 `1` 和 `2` 弹幕我倒不 care，我个人不会删除弹窗。这个 discussion 就只是希望能优化下弹窗。

**Reply by Geeyun-JY3** [reply]
没怎么留意什么时候改的，现在好像出来的投票弹幕好像已经是选项的文字了。
